docs/why-was-fakeiteasy-created.md

```diff
@@ -15,7 +15,7 @@ The question on Stack Overflow:
 #Patrik H&auml;gne's answer
 
 To be clear, I created FakeItEasy so I'll definitely not say whether
-one framework is better than the other, what I can do is point out

```

Comment:
I hesitate to change this word, and the one on line 48, since this text actually quotes M. Hägne's answer on StackOverflow.

Reply:
Ah right, I can imagine. Can change those back if you like.

Reply:
I agree. Better to keep Patrik's original text unchanged.